src/pseudopeople/utilities.py

```diff
@@ -16,9 +17,23 @@ def get_default_configuration() -> ConfigTree:
 
     default_config_layers = [
         "base",
+        "user",
     ]
     noising_configuration = ConfigTree(layers=default_config_layers)
```

Comment:
Take a look at the code for the `ConfigTree` init function. You can set the base layer directly in the init without a need for a separate `update` call.